samples/app/src/main/kotlin/com/bumble/appyx/app/node/teaser/backstack/BackstackTeaserNode.kt

```diff
@@ -36,18 +36,20 @@ class BackstackTeaserNode(
 
     init {
         lifecycle.coroutineScope.launch {
-            delay(1000)
-            repeat(4) {
-                backStack.push(NavTarget.Child((it + 2) * 100))
-                delay(400)
+            lifecycle.coroutineScope.launchWhenStarted {
```

Comment:
Why do we need to use both `launch` and `launchWhenStarted`? Just `launchWhenStarted` should be enough, it is not suspending.

Reply:
This is the recommend approach from Google, the other approach is deprecated.

This code also ensures that it stops the coroutine when I stop happens

Reply:
Oh actually, I'll double check this, I meant to use a different method. I might revert this for this PR, and add the other change later

Reply:
Fixed.